integration/specs/TimePicker/timePicker-1.spec.js

```diff
@@ -0,0 +1,576 @@
+        picklist.waitUntilOpen();
+        browser.keys(ARROW_RIGHT_KEY);
+        picklist.clickUpButton();
+        picklist.clickUpButton();
+        picklist.clickUpButton();
+        expect(picklist.getMinutesValue()).toBe('02');
+    });
+    it('should set the right minutes input value and focus am-pm selector when type "6" and minutes input is focused', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        browser.keys(ARROW_RIGHT_KEY);
+        browser.keys('6');
+        expect(picklist.hasFocusAmPmSelect()).toBe(true);
+        expect(picklist.getMinutesValue()).toBe('06');
+    });
+    it('should set the right minutes input value and focus am-pm selector when type "09" and minutes input is focused', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        browser.keys(ARROW_RIGHT_KEY);
+        browser.keys('0');
+        browser.keys('9');
+        expect(picklist.hasFocusAmPmSelect()).toBe(true);
+        expect(picklist.getMinutesValue()).toBe('09');
+    });
+    it('should set the right minutes input value and focus am-pm selector when type "56" and minutes input is focused', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        browser.keys(ARROW_RIGHT_KEY);
+        browser.keys('5');
+        browser.keys('6');
+        expect(picklist.hasFocusAmPmSelect()).toBe(true);
+        expect(picklist.getMinutesValue()).toBe('56');
+    });
+    it('should set the right minutes input value and keep the focus on minutes input when type "5" and minutes input is focused', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        browser.keys(ARROW_RIGHT_KEY);
+        browser.keys('5');
+        expect(picklist.hasFocusMinutesInput()).toBe(true);
+        expect(picklist.getMinutesValue()).toBe('05');
+    });
+    it('should set the right minutes input value and keep the focus on minutes input when type "3" and minutes input is focused', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        browser.keys(ARROW_RIGHT_KEY);
+        browser.keys('3');
+        expect(picklist.hasFocusMinutesInput()).toBe(true);
+        expect(picklist.getMinutesValue()).toBe('03');
+    });
+    it('should set the right minutes input value and keep the focus on minutes input when type "0" and minutes input is focused', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        browser.keys(ARROW_RIGHT_KEY);
+        browser.keys('0');
+        expect(picklist.hasFocusMinutesInput()).toBe(true);
+        expect(picklist.getMinutesValue()).toBe('00');
+    });
```

Comment:
in a previous test you type 0 and then press down
so remove this test and add this expects in that test